fe/fe-core/src/main/java/org/apache/doris/common/util/ProfileManager.java

```diff
@@ -104,160 +90,60 @@ public static ProfileManager getInstance() {
     }
 
     private ProfileManager() {
-        lock = new ReentrantReadWriteLock(true);
-        readLock = lock.readLock();
-        writeLock = lock.writeLock();
-        queryIdDeque = new LinkedList<>();
-        queryIdToProfileMap = new ConcurrentHashMap<>();
+        storage = new InMemoryProfileStorage();
```

Comment:
Will we use variables to decide which storage to use in the future?

Reply:
@stalary yes, I plan to use  variables in next pr, because there is only one impl of it